src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -371,6 +372,25 @@ public PackageSearchState SearchState
                 RaisePropertyChanged(nameof(this.SearchState));
                 RaisePropertyChanged(nameof(this.SearchBoxPrompt));
                 RaisePropertyChanged(nameof(this.ShowSearchText));
+
+                if (value == PackageSearchState.Results && !this.InitialResultsLoaded)
+                {
+                    this.InitialResultsLoaded = true;
+                }
+            }
+        }
+
+        private bool _initialResultsLoaded = false;
+        /// <summary>
+        /// Will only be set to true once after the initial search has been finished.
+        /// </summary>
+        public bool InitialResultsLoaded
```

Comment:
Does it need to be public or internal? Same questions to other new properties, our team is trying to reduce `public` exposed method or properties these days

Reply:
This one follows the typical wpf MVVM pattern and it the view won't be updated if it is internal :/ 
 
```
private bool _initialResultsLoaded = false;
/// <summary>
/// Will only be set to true once after the initial search has been finished.
/// </summary>
public bool InitialResultsLoaded
{
    get { return _initialResultsLoaded; }
    set
    {
        _initialResultsLoaded = value;
        RaisePropertyChanged(nameof(InitialResultsLoaded));
    }
}
```